setup.cfg

```diff
@@ -1,6 +1,12 @@
 [pycodestyle]
 count = True
-ignore=E121,E123,E126,E133,E226,E241,E242,E704,W503,W504,W505,E741,W605
+ignore=E121,E123,E126,E133,E226,E241,E242,E704,W503,W504,W505,E741,W605,E203
```

Comment:
What is E203?

Reply:
black (which is called by the pre-commit stack defined in the yaml file above) reformat the white space around ":" correctly but pycodetyle returns a false positive.

https://github.com/psf/black/issues/279
https://github.com/PyCQA/pycodestyle/issues/373